workspace/scm.py

```diff
@@ -193,6 +193,15 @@ def merge_branch(branch, squash=False, strategy=None):
         cmd.append('--squash')
     if strategy:
         cmd.append('--strategy=' + strategy)
+
+    current = current_branch()  # uses current_branch() method  to get current branch
```

Comment:
i don't think you need that comment since its pretty clear based on the method name, but good to see you commenting your code.